frontend/integration-tests/views/storage.view.ts

```diff
@@ -0,0 +1,17 @@
+import { $, browser, ExpectedConditions as until } from 'protractor';
+import * as crudView from '../views/crud.view';
+
+export const createNewClaim = $('[value=new]');
+export const inputPVCName = $('#pvc-name');
+export const inputPVCSize = $('[name=requestSizeValue]');
+export const inputMountPath = $('#mount-path');
+
+export const addNewStorageToWorkload = async function(pvcName:string, pvcSize:string, mountPath:string) {
```

Comment:
If this is only used in the storage scenario, put this there.

I'd just call this `addStorageToWorkload`

nit: space after `:`

```suggestion
export const addStorageToWorkload = async function(pvcName: string, pvcSize: string, mountPath: string) {
```

Reply:
I name it "addNewStorageToWorkload" is to distinguish with another scenario: add existing storage to workload.  Though the latter scenario is not added yet in the repo.

> If this is only used in the storage scenario, put this there.
Are you meaning putting it in integration-tests/tests/storage.scenario.ts?